### DISCUSSION TITLE
Recommended way to upgrade SABnzbd

### DISCUSSION THREAD

**Discussion by sendmebits** [tteck/Proxmox#1377] [Help Needed] [answered]
Is there a recommended way to programmatically update SABnzbd?

I assumed the "**Proxmox VE LXC Updater**" script would do it, but it just seems to update the LXC and not the SABnzbd app.

I did see a past discussion on this topic where it seemed like the outcome was that the script would do it? My SABnzbd is currently 3.7.2 while 4.0.x has recently released. 

https://github.com/tteck/Proxmox/discussions/1199

**Comment by tteck** [comment, marked as answer]
![Screenshot 2023-04-30 5 18 20 PM](https://user-images.githubusercontent.com/21342670/235376707-359c28e5-4a37-41b1-85f6-9bc4b9b6de58.png)

**Reply by sendmebits** [reply]
Ugh, completely missed that.  Works like a charm, thank you sir!
